json_schemas/auth_users.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "API Updater Schema",
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "email_address": {
+      "type": "string",
+      "pattern": "^[^@^\\s]+@[^@^\\.^\\s]+(\\.[^@^\\.^\\s]+)+$"
```

Comment:
JSONSchema has some built in formats of which email is one. It's not highlighted very well.
http://spacetelescope.github.io/understanding-json-schema/reference/string.html#format
And the Python library does support format checking
http://python-jsonschema.readthedocs.org/en/latest/validate/#validating-formats